js/deer-render.js

```diff
@@ -1043,70 +1043,249 @@ DEER.TEMPLATES.managedlist = function (obj, options = {}) {
     try {
         // If the collection doesn't have a name, something has gone wrong.
         if(!obj.name) return
-        let tmpl = ` `
+        let tmpl = ` 
+        <style>
+            .cachedNotice{
+            margin-top: -1em;
+            display: block;
+            }
+
+            .cachedNotice a{
+            cursor: pointer;
+            }
+
+            .totalsProgress{
+            text-align: center;
+            background-color: rgba(0, 0, 0, 0.1);
+            padding-top: 4px;
+            font-size: 13pt;
+            }
+        </style>
+        <h2> Glosses </h2>
+        <small class="cachedNotice is-hidden text-primary"> These Glosses were cached.  To reload the data <a class="newcache tag is-small">click here</a>. </small>
+        <input filter="title" type="text" placeholder="&hellip;Type to filter by incipit, text, or targeted text" class="is-hidden serifText">
+        <div class="progressArea">
+            <p class="filterNotice is-hidden"> Gloss filter detected.  Please note that Glosses will appear as they are fully loaded. </p>
+            <div class="totalsProgress" count="0"> {loaded} out of {total} loaded (0%).  This may take a few minutes.  You may click to select any Gloss loaded already.</div>
+        </div>`
+        let managedListCache = localStorage.getItem("managedListCache") ? new Map(Object.entries(JSON.parse(localStorage.getItem("managedListCache")))) : new Map()
```

Comment:
You need to use the same localStorage entry as the other templates, otherwise they won't share the cache. All the gloss list related templates can use the same cache.